erpc_c/port/erpc_port_zephyr.cpp

```diff
@@ -54,13 +54,13 @@ void operator delete[](void *ptr) THROW
 
 void *erpc_malloc(size_t size)
 {
-    void *p = k_malloc(size);
+    void *p = malloc(size);
```

Comment:
FWIW, I am using k_malloc/k_free here without any issues.

Reply:
The specific issue I was having was that I would allocate a buffer with a normal `malloc` in the implementation of `echo` and it would crash when erpc would try to free that buffer.

I'm ok with switching it back, but I would assume that people extending the API are most likely to be returning buffers created from a normal `malloc` rather than `erpc_malloc` or a `k_malloc`.